src/interpreter/runtime/values/number.js

```diff
@@ -69,6 +70,92 @@ export class BaseNumber implements NumberInterface<BaseNumber> {
     return new BaseNumber(-this._value);
   }
 
+  /*eslint-disable no-unused-vars*/
```

Comment:
Just FYI I commented the operand somewhere in a PR, like:
```js
rem_s(/*operand: BaseNumber*/): BaseNumber {
    throw new RuntimeError("Unsupported operation");
}
```

Reply:
Ah, good point - for some reason I thought doing that would result in Flow errors, but it does not :-)